packages/office-ui-fabric-react/src/components/__snapshots__/ChoiceGroup.Basic.Example.tsx.shot

```diff
@@ -4,7 +4,7 @@ exports[`Component Examples renders ChoiceGroup.Basic.Example.tsx correctly 1`]
 <div>
   <div
     className=
-
+        defaultChoiceGroup
```

Comment:
I kind of would have expected to see a snapshot change where `label` used to have className but no longer does. I imagine this means we lack such a test case. It would be great if you can add it.